pennylane_lightning/core/src/simulators/lightning_gpu/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+    LANGUAGES CXX C CUDA
+)
+
+# Include macro and functions supporting Nvidia and cuQuantum libraries.
+include("${pennylane_lightning_SOURCE_DIR}/cmake/support_pllgpu.cmake")
+findCUDA(lightning_external_libs)
+findCustatevec(lightning_external_libs)
+
+if(ENABLE_MPI)
+    findMPI(lightning_external_libs)
+endif()
+
+set(LGPU_FILES  StateVectorCudaManaged.cpp initSV.cu
+                CACHE INTERNAL "" FORCE)
+
+
+add_library(${PL_BACKEND} STATIC ${LGPU_FILES})
+
+target_compile_options(lightning_compile_options INTERFACE "-D_ENABLE_PLGPU=1")
+
+if(ENABLE_MPI)
+    target_compile_options(lightning_compile_options INTERFACE "-D_ENABLE_MPI=1")
+endif()
+
+##########################
+## Enforce C++ Standard ##
+##########################
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+# Ensure the libraries can see additional libs at same level;
+# Required for external deps when loading in Python
+set(CMAKE_BUILD_RPATH_USE_ORIGIN ON)
+set(CMAKE_SKIP_BUILD_RPATH FALSE)
+set(CMAKE_BUILD_RPATH "$ORIGIN/../cuquantum/lib:$ORIGIN/../cuquantum/lib64:$ORIGIN/")
+set(CMAKE_INSTALL_RPATH "$ORIGIN/../cuquantum/lib:$ORIGIN/../cuquantum/lib64:$ORIGIN/")
+
+#########################
+## Set Default Options ##
+#########################
+option(PLLGPU_ENABLE_SANITIZER "Enable address sanitizer" OFF)
+option(PLLGPU_ENABLE_WARNINGS "Enable warnings" ON)
+option(PLLGPU_DISABLE_CUDA_SAFETY "Build without CUDA call safety checks" OFF)
+
+
+if(PLLGPU_ENABLE_SANITIZER)
+    add_compile_options(-fsanitize=address)
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+        add_link_options(-fsanitize=address)
+    else ()
+        add_link_options(-fsanitize=address -static-libasan)
+    endif()
+    set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fno-omit-frame-pointer -fsanitize=address")
+    set (CMAKE_LINKER_FLAGS_DEBUG "${CMAKE_LINKER_FLAGS_DEBUG} -fno-omit-frame-pointer -fsanitize=address")
+ENDIF()
+
+target_link_libraries(${PL_BACKEND} PUBLIC  lightning_compile_options
+                                            lightning_external_libs
+                                            lightning_base
+                                            lightning_gates
+                                            lightning_utils
+                                            ${PL_BACKEND}_utils
+                                            ${PL_BACKEND}_gates
+                                            )
+    
+target_include_directories(${PL_BACKEND} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+set_property(TARGET ${PL_BACKEND} PROPERTY POSITION_INDEPENDENT_CODE ON)
+
+# To avoid DSO errors on platforms preferring static linkage, uncomment the following line:
+# string(REPLACE "libcudart_static.a" "libcudart.so" CUDA_SHARED_RT "${CUDA_LIBRARIES}")
+set_target_properties(${PL_BACKEND} PROPERTIES INSTALL_RPATH "$ORIGIN/../cuquantum/lib:$ORIGIN/../cuquantum/lib64:$ORIGIN/")
```

Comment:
As mentioned earlier about the RPATH. Fine if we can validate things haven't changed.

Reply:
It work fine for both cu-11 and cu-12.